src/variorum/Intel_GPU/config_intel_gpu.c

```diff
@@ -26,7 +26,8 @@ int set_intel_gpu_func_ptrs(int idx)
         g_platform[idx].variorum_print_power           = intel_gpu_get_power;
         g_platform[idx].variorum_print_thermals        = intel_gpu_get_thermals;
         g_platform[idx].variorum_print_frequency       = intel_gpu_get_clocks;
-        // g_platform.variorum_print_power_limit    = ; // implement this later
+        g_platform[idx].variorum_print_power_limit     = intel_gpu_get_power_limit;
```

Comment:
In addition to the new API for cap_gpu_power limit, this PR also implements the print power limit for Intel DGPU (not a new API, but missing functionality on Intel DGPU) -- can you update the docs to reflect this?
My guess is @slabasan would want this to be a separate PR, I am ok either way, as in, we can merge it in with this PR.

Reply:
Ah I didn't catch that, yes would prefer if that went into another PR. How much of the print power limit for Intel DGPU implementation is in this PR? Docs and implementation? Just implementation?

Reply:
I just saw the implementation, I didn't see the associated docs for this in this PR. I think removing from this one and adding that in another PR is a good idea. Tagging @amarathe84 so he sees it